.github/workflows/grumphp.yml

```diff
@@ -0,0 +1,33 @@
+name: GrumPHP
+
+on: [push]
+jobs:
+    run:
+        runs-on: ${{ matrix.operating-system }}
+        strategy:
+            matrix:
+                operating-system: [ubuntu-latest, macos-latest, windows-latest]
+                php-versions: ['7.2', '7.3', '7.4']
+                composer-options: ['', '--prefer-lowest']
+        name: PHP ${{ matrix.php-versions }} @ ${{ matrix.operating-system }} ${{ matrix.composer-options }}
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v1
+            - name: Install PHP
+              uses: shivammathur/setup-php@master
+              with:
+                  php-version: ${{ matrix.php-versions }}
+                  extensions: xdebug, mbstring #optional
+            - name: Check PHP Version
+              run: php -v
+            - name: Validate composer.json and composer.lock
+              run: composer validate
+            - name: Install dependencies
+              run: composer update --prefer-dist --no-progress --no-suggest ${{ matrix.composer-options }}
+            - name: Run non windows test suite
+              run: vendor/bin/grumphp run
+              if: matrix.operating-system != 'windows-latest'
+            - name: Run windows test suite
+              run: vendor/bin/grumphp run --tasks phpunit,paratest,phpspec
```

Comment:
Currently the complete testsuite is being validated on windows as well through appvoyer.
I would prefer to have all tasks tested. Why are you limiting it here?

Reply:
Think i just replicated what there was.

Reply:
No, we are running everything on windows. See : https://github.com/phpro/grumphp/blob/master/appveyor.yml#L74